From abf8c4bf53a84ef3de530519a3bbb6b599595f2c Mon Sep 17 00:00:00 2001
From: robdewit <rdewit@wise-guys.nl>
Date: Tue, 31 Mar 2015 14:24:15 +0200
Subject: [PATCH] Update main.c

Removed bug where cron.update is not picked up while jobs are still running.
---
Fix backported from upstream master - https://github.com/dubiousjim/dcron/commit/abf8c4bf53a84ef3de530519a3bbb6b599595f2c
---
 main.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff -upr dcron-4.5.orig/main.c dcron-4.5/main.c
--- dcron-4.5.orig/main.c	2020-03-27 15:46:16.344036288 +0700
+++ dcron-4.5/main.c	2020-03-27 15:43:29.164619553 +0700
@@ -343,7 +343,8 @@ main(int ac, char **av)
 					SynchronizeDir(SCDir, "root", 0);
 					ReadTimestamps(NULL);
 				}
-			} else {
+			}
+			if (rescan < 60) {
 				CheckUpdates(CDir, NULL, t1, t2);
 				CheckUpdates(SCDir, "root", t1, t2);
 			}
Only in dcron-4.5: main.c.orig
